backend/src/main/resources/db/changelog/db.changelog-v3.yaml

```diff
@@ -90,4 +90,51 @@ databaseChangeLog:
         - modifyDataType:
             tableName: SUBMODEL_ENDPOINT
             columnName: SUB_PROTOCOL_BODY_ENCODING
-            newDataType: NVARCHAR(128)
\ No newline at end of file
+            newDataType: NVARCHAR(128)
```

Comment:
This will be failed. You can not add new entries in existing changeset. 
Please create a new changeset and add your changes.

Reply:
incorporated the review comment.